homebrew/reddit/goat-background.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Goat</name>
+		<description>Goat Background</description>
+		<author url="https://www.reddit.com/r/UnearthedArcana/comments/7jnwr5/was_a_goat_for_10_years/">u/_Metabot</author>
+		<update version="0.0.1">
+			<file name="goat-background.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/homebrew/reddit/goat-background.xml" />
+		</update>
+	</info>
+	
+	<element name="Was a Goat for 10 Years" type="Background" source="Reddit" id="ID_BACKGROUND_GOAT">
```

Comment:
even though i don't think there will be many goat backgrounds, please change IDs for background/feature to include author/source to make it more unique (`ID_RDDT`)

Reply:
Changed to "ID_RDDT_BACKGROUND_GOAT"